.travis.yml

```diff
@@ -1,14 +1,17 @@
 sudo: false
-dist: trusty
+dist: xenial
 
 notifications:
   email: false
 
 language: python
-python: 3.6
+python:
+  - 3.5
+  - 3.6
+  - 3.7
 cache: pip
 
 install:
-  - pip install tox
+  - pip install tox tox-travis
 
-script: tox
+script: tox -r
```

Comment:
`-r` is not needed on travis since it's a fresh environment each time, `.tox` is not cached

Reply:
Ok, its just a little habit of mine :) I'll remove it :)